r-package/grf/R/plot.R

```diff
@@ -93,3 +97,76 @@ plot.grf_tree <- function(x, ...) {
   dot_file <- export_graphviz(x)
   DiagrammeR::grViz(dot_file)
 }
+
+leaf_stats <- function(forest, samples) UseMethod("leaf_stats")
```

Comment:
I think we should move these methods to `analysis_tools.R`, since now they are invoked within `get_tree`.

Reply:
I was thinking the same thing- sounds good!